ibis/backends/tests/test_aggregation.py

```diff
@@ -195,27 +199,67 @@ def test_aggregate_grouped(
             lambda t, where: t.double_col.cov(t.float_col),
             lambda t, where: t.double_col.cov(t.float_col),
             id='covar',
+            marks=[
+                pytest.mark.backends_notimpl(
+                    [
+                        "clickhouse",
+                        "csv",
+                        "dask",
+                        "hdf5",
+                        "impala",
+                        "mysql",
+                        "pandas",
+                        "parquet",
+                        "postgres",
+                        "pyspark",
+                        "sqlite",
+                    ]
+                )
+            ],
         ),
         param(
             lambda t, where: t.double_col.corr(t.float_col),
             lambda t, where: t.double_col.corr(t.float_col),
             id='corr',
+            marks=[
+                pytest.mark.backends_notimpl(
+                    [
+                        "clickhouse",
+                        "csv",
+                        "dask",
+                        "hdf5",
+                        "impala",
+                        "mysql",
+                        "pandas",
+                        "parquet",
+                        "postgres",
+                        "pyspark",
+                        "sqlite",
+                    ]
```

Comment:
So only the `datafusion` backend implement the `corr` and `cov` reductions?

Reply:
This is fine as is, just wanted to make sure that it wasn't being lost in the sea of backends here 😬